lib/gitlab_custom_hook.rb

```diff
@@ -48,11 +48,14 @@ def call_receive_hook(hook, changes)
       # need to close stdin before reading stdout
       stdin.close
 
+      stdout_stderr.each_line do |line| 
+        puts line 
```

Comment:
Trailing whitespace detected.<br>Do not write to stdout. Use Rails' logger if you want to log.